### DISCUSSION TITLE
HTTP/3 and NodeJS 18 support

### DISCUSSION THREAD

**Discussion by SergeySypalo** [vercel/vercel#8823] [Ideas] [answered]
Hi there,
After checking official docs, where stated that the latest NodeJS LTS is supported, I don't see version 18 in my project settings NodeJS selector dropdown list, only 14 and 16.
Also I would like to ask to add HTTP/3 support for even faster speed as it adds some great improvements: https://en.wikipedia.org/wiki/HTTP/3

**Comment by swarnava** [comment, marked as answer]
Hi there,

We don't support Node 18 yet. You can see currently supported version here: https://vercel.com/docs/runtimes#official-runtimes/node-js/node-js-version. We will make [announcement](https://vercel.com/changelog) once this is ready.

HTTP/3 is currently not supported.

**Reply by SergeySypalo** [reply]
Any timelines?

**Reply by swarnava** [reply]
Once our upstream provider start supporting it, we will make them available.

For HTTP/3, unfortunately we don't have a timeline.

**Reply by SergeySypalo** [reply]
Gotcha, thanks for the quick reply.

**Reply by DylanVann** [reply]
Which provider is being waited on for HTTP3? CloudFront and Cloudflare both support HTTP3

**Comment by LarsArtmann** [comment]
HTTP/3 would be great, hope it will be added to your road map soon.

**Comment by ornakash** [comment]
We really need this!
